src/main/java/com/snowflake/kafka/connector/internal/streaming/BufferedTopicPartitionChannel.java

```diff
@@ -402,13 +402,13 @@ public void insertRecord(SinkRecord kafkaSinkRecord, boolean isFirstRowPerPartit
         insertRecords(copiedStreamingBuffer);
       }
     } else {
-      LOGGER.debug(
-          "Skip adding offset:{} to buffer for channel:{} because"
-              + " offsetPersistedInSnowflake:{}, processedOffset:{}",
-          kafkaSinkRecord.kafkaOffset(),
+      LOGGER.warn(
+          "Channel {} expected record at offset {} but received {}. The current offset stored in"
```

Comment:
Thank you very much for addressing this!

Could we keep the skip messaging too (my suggested change in Slack was meant to be additive, sorry for the confusion!)

The new warn message is great, but it doesn't let the reader know what the connector's action was (skip) so they can mentally match it to their observed lag/loss situation.

Reply:
ok, extended the message a bit.